### PR TITLE
Add darker

### PR DESCRIPTION
**Pull request by martinRenou**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by martinRenou** [comment]
@akaihola I've added you as the committer, as I realised later that you started working on this on your side, I don't want to be stealing credit from you.

Please confirm that you are willing to maintain this conda package :)

**Comment by akaihola** [comment]
Hi @martinRenou,

I'd be happy to see someone else take responsibility for maintaining the conda package! So go ahead with it, no need to give attribution to me. Thanks for your effort!

For reference, akaihola/darker#114 was the original issue for creating a conda package.

File: recipes/darker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - dataclasses        # [py<37]
+
+test:
+  imports:
+    - darker
+  requires:
+    - pip
+  commands:
+    - pip check
+    - test -f ${PREFIX}/bin/darker
```
Comment (chrisburr) [inline recipes/darker/meta.yaml:38]: Can you use something like this instead for testing?
```suggestion
    - darker --help
```

File: recipes/darker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3a840c2cedec7e4d13c26c450d94f31bd872375450a0c057b9276cc9c2dbad5c
+
+build:
+  # noarch: python
+  skip: True  # [win]
```
Comment (chrisburr) [inline recipes/darker/meta.yaml:14]: What's the issue with Windows? Was it just that `test -f` didn't work?

Comment (martinRenou) [reply recipes/darker/meta.yaml:14]: There is currently an installation issue on Windows https://github.com/akaihola/darker/pull/120 

At some point we might be able to make this package noarch, but due to this Windows issue we can't for now

**Comment by martinRenou** [comment]
Thanks for the review @chrisburr !

**Comment by akaihola** [comment]
The Windows installation issue was fixed in akaihola/darker#121 (merged) and akaihola/darker#124 (pending review). These fixes will be included in the next release (version 1.2.3).

File: recipes/darker/meta.yaml (hunk 2 above)
Comment (akaihola) [reply recipes/darker/meta.yaml:14]: akaihola/darker#120 has been merged and included in later versions – has this skip been removed?

Comment (martinRenou) [reply recipes/darker/meta.yaml:14]: Yes it has been removed, you can see it here that the package is now noarch https://github.com/conda-forge/darker-feedstock/blob/main/recipe/meta.yaml#L13

**Pull request merged**